packages/@aws-cdk/aws-pipes-sources-alpha/README.md

```diff
@@ -61,3 +61,22 @@ const pipe = new pipes.Pipe(this, 'Pipe', {
 });
 ```
 
+### Amazon Kinesis
+
+A Kinesis stream can be used as a source for a pipe. The stream will be polled for new messages and the messages will be sent to the pipe.
+
+```ts
+import * as kinesis from 'aws-cdk-lib/aws-kinesis';
```

Comment:
I'm not sure why this is required but an import for SQS isn't needed ?

Reply:
They are declared in the default rosetta file 
https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-pipes-sources-alpha/rosetta/default.ts-fixture